src/lib/token.h

```diff
@@ -50,20 +50,39 @@ struct sealobject {
     uint32_t handle;
 };
 
+enum token_type {
+    token_type_esysdb = 0,
+    token_type_fapi
+};
+
 typedef struct token token;
 struct token {
 
     unsigned id;
     unsigned pid;
     unsigned char label[32];
 
-    token_config config;
+    enum token_type type;
 
-    pobject pobject;
+//TODO: uncomment union, once both backends are completely separated.
+//    union { /* anon union for backend data */
```

Comment:
I thought we we're merging the backends, so you can have N ESYS DB tokens + M FAPI tokens?